core/app/models/concerns/spree/default_price.rb

```diff
@@ -32,6 +32,11 @@ def default_price_or_build
         prices.build(self.class.default_price_attributes)
     end
 
+    def find_or_build_default_price
+      default_price_or_build
+    end
+    deprecate :find_or_build_default_price, deprecator: Spree::Deprecation
```

Comment:
```suggestion
    deprecate :find_or_build_default_price, :default_price_or_build, deprecator: Spree::Deprecation
```

👆 I think this will generate a better deprecation message.

Reply:
We're trying to get better with yard documentation, so here we have a nice opportunity to use the `@deprecated` tag: https://rubydoc.info/gems/yard/file/docs/Tags.md#deprecated :slightly_smiling_face: